irmin-git.opam

```diff
@@ -18,7 +18,7 @@ depends: [
   "dune"       {>= "2.7.0"}
   "irmin"      {= version}
   "ppx_irmin"  {= version}
-  "git"        {>= "2.1.1"}
+  "git"
```

Comment:
any reason to remove the constraint here?

Reply:
Will be `>= 3.0.0` at the release of `ocaml-git`.